### ISSUE TITLE
Impossible coverages when combining regional and global queries

### ISSUE THREAD

**Issue by anssih** [ai/browserslist#61]
````
$ browserslist --coverage=FI "> 1%, > 1% in FI" 
These browsers account for 105.31% of all users in the FI
````

Coverage over 100% seems impossible, especially because the total caniuse-db coverage in FI.json seems to be 93.63%.

The same query with US also seems to gives wrong result, though not as apparent:
````
$ browserslist --coverage=US "> 1%, > 1% in US" 
These browsers account for 98.93% of all users in the US
````
while the total coverage in region-usage-json/US.json is only 95.96%.

This is with browserslist 1.3.2 with caniuse-db 1.0.30000484.

**Comment by ai** [comment]
:D strange

**Comment by ai** [comment]
I found a problem. Browsers selection contains `and_chr 0` and `and_chr 50`

**Comment by ai** [comment]
Fixed f8bb113

**Comment by ai** [comment]
Released in 1.3.3
